sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -69,10 +72,14 @@ def _get_instance_types(region: str) -> pd.DataFrame:
 
 
 @ray.remote
-def _get_availability_zones(region: str) -> pd.DataFrame:
-    client = aws.client('ec2', region_name=region)
-    zones = []
-    response = client.describe_availability_zones()
+def _get_availability_zones(region: str) -> Optional[pd.DataFrame]:
+    try:
+        client = aws.client('ec2', region_name=region)
+        zones = []
+        response = client.describe_availability_zones()
```

Comment:
Should we only try-catch this? https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/boto3.html & @iojw suggest `aws.client()` above doesn't raise.

Reply:
Good point! Moved. Thanks!